packages/api/src/functions/graphql.ts

```diff
@@ -8,7 +8,7 @@ import type { Config, CreateHandlerOptions } from 'apollo-server-lambda'
 import type { ApolloServerPlugin } from 'apollo-server-plugin-base'
 import type { APIGatewayProxyEvent, Context as LambdaContext } from 'aws-lambda'
 import depthLimit from 'graphql-depth-limit'
-import { BaseLogger, LevelWithSilent } from 'pino'
+import pino from 'pino'
```

Comment:
Would importing types from https://github.com/pinojs/pino/blob/master/pino.d.ts allow use of `.LevelWithSilent` vs `pino.LevelWithSilent` -- wondering if `pino.*` will be seen in docs/help.

Reply:
I was trying to solve all the TS error after update because `import { BaseLogger, LevelWithSilent } from 'pino'` didn't work anymore. Using `pino.LevelWithSilent` solved the problem, but I agree with you that it is better to just have the type definition. How can I just import the `pino.d.ts` ?

Reply:
Since they wrap all type inside `namespaces P` we cannot import each type individually.  
So I was like why not use destructuring... but it not possible for types yet 😬
https://github.com/microsoft/TypeScript/issues/13135

So the only solution if we don't want the `P.*` or `pino.*` before type is to use Aliases, one by one.
https://www.typescriptlang.org/docs/handbook/namespaces.html#aliases

Reply:
> So the only solution if we don't want the P.* or pino.* before type is to use Aliases, one by one.

@dac09 What is you take on use of the namespaces ... or should we hide the types in some way behind a redwood set of logger types and not expose Pino in the app?